mlibrary/lib.umich.edu
# Changing twitter to X

# Overview
Changing "Twitter" to "X" as part of the univeristy-wide social update.

In instances where "X" is used alone, we are also adding "(formerly Twitter)".

We're creating the SVG icons ourselves instead of using a library like Font Awesome - React. I created the "X" logo in Illustrator from the Font Awesome icon to fit the style and size as the other social media icons.

> This pull request resolves/closes/fixes [WEBSITE-111](https://mlit.atlassian.net/jira/software/projects/WEBSITE/boards/27?selectedIssue=WEBSITE-111).

## Anything else?
No errors in WAVE / Axe DevTools.

The icon on the profile page is still reading "Twitter" for screen readers. I believe this is being pulled from the staff data and not something we're setting on our end. 

## Testing
Pages / components where icon has changed:
- [Footer component on all pages](https://future-wwwlib-previews.netlify.app/)
- [Share component on News and Events templates](https://future-wwwlib-previews.netlify.app/about-us/news/library-launches-campus-wide-survey)
- [Profile template where employee has Twitter linked](https://future-wwwlib-previews.netlify.app/users/bnhowell)

---
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
  - [x] NVDA (I had some issues with the new NVDA update. If you get an error saying "Legacy Browser" after updating, restarting my machine fixed the error.)
